static/js/pages/profiles/ProfileSelector.tsx

```diff
@@ -77,10 +72,14 @@ const ProfileSelector: FC<Props> = ({
 
   return (
     <>
-      <Label className="u-sv-1">Apply profiles</Label>
-      <div className="u-text--muted u-sv1">
-        Add profiles and create a hierarchy
-      </div>
+      <Row>
+        <Col size={12}>
+          <Label className="u-sv-1">Apply profiles</Label>
```

Comment:
Do we need `label` here? If we need only the styles, maybe we could only apply needed styles or use existing Vanilla class?

Reply:
This is a WIP component, that will fundamentally change when the design is ready. Currently we need the label as the rest of the component is custom.